src/lro/models.ts

```diff
@@ -0,0 +1,38 @@
+import { OperationArguments, OperationSpec } from "@azure/core-http";
+import { PollOperationState, PollOperation } from "@azure/core-lro";
+
+export interface BaseResult {
+  location?: string;
+  operationLocation?: string;
+  azureAsyncOperation?: string;
+  provisioningState?: string;
+  properties?: {
+    provisioningState?: string;
```

Comment:
Is `provisioningState` an example specifically for our `lro.json` swagger, or is it well-defined as something that will always be present when we attempt the "BodyPolling" strategy?

Reply:
provisioningState is standard for BodyPolling strategy, I think it is a property regularly used for Azure Resource management libraries.

Reply:
Gotcha. Actually really glad to hear that there’s a standard field being used for this case!